### PR TITLE
Placement of ship method

### PR DESCRIPTION
**Pull request by melatran**
I made a test to place the ships on the board as well as another test on not overlapping the ships. I'm not quite sure if I wrote that test out correctly.

### CODE REVIEW

File: test/board_test.rb [hunk 1]
```diff
@@ -56,4 +56,23 @@ def test_can_place_ship
...
+    cell_2 = @board.cells["A2"]
+    cell_3 = @board.cells["A3"]
+
+    assert_equal @cruiser, cell_1.ship
+    assert_equal @cruiser, cell_2.ship
+    assert_equal @cruiser, cell_3.ship
+    assert_equal true, cell_3.ship == cell_2.ship
+  end
+
+  def test_can_not_overlap_ships_on_board
```
Comment (melatran) [inline test/board_test.rb:73]: Please check if this test is written correctly. It passes but I'm not sure if I'm missing anything.

File: lib/board.rb [hunk 2]
```diff
@@ -69,4 +69,10 @@ def valid_vertical_coordinates?(coordinates)
   def can_place_ship?(coordinates)
     valid_horizontal_coordinates?(coordinates) || valid_vertical_coordinates?(coordinates)
   end
+
+  def place(ship, coordinates)
+    coordinates.each do |coordinate|
+        @cells[coordinate].place_ship(ship)
```
Comment (melatran) [inline lib/board.rb:75]: note to myself:
hash[key] access the value using this (the key is the coordinate)
in this method, ruby is grabbing the value (cell id) using the coordinate from @board.cells ['A1']

File: test/board_test.rb (hunk 1 above)
Comment (whitneykidd) [reply test/board_test.rb:73]: This looks correct to me - You've tested .place, .ship. and .valid_placement and edge cases.  👍
